### DISCUSSION TITLE
Marking already converted books

### DISCUSSION THREAD

**Discussion by rbielaws** [audiamus/BookLibConnect#120] [Q&A] [answered]
I've had Audible for many years and have already downloaded/converted several hundred books with other software.
As I switch to this (+ AAX audio converter) I think it would be terribly convenient if I could flag the hundreds of books already converted so they don't constantly appear to need attention.

If nothing already exists, I'm rather adept at database manipulation and have the SQLite CLI available.  Pasting some table update commands (for example, something against Conversions.State??) or whatever you think will do the trick is just as useful to me.

Thanks.

**Comment by rbielaws** [comment, marked as answer]
Well, despite the comment "I would also urgently recommend not to modify any data." (in reply to another user's question) I did root around in the source and decided setting the Conversions.State value to 14 (converted_unknown) probably couldn't hurt.  After a couple tests I set the remaining rows like this (update conversions set state=14 where state=1;).  Now all those older book rows are distinctly out of the way of being mixed up with new books.

If I did something foolishly wrong, please set me straight so others don't follow me over a cliff.

By the way, absolutely great work!  
Thanks for such an amazingly complete and useful tool.

**Comment by audiamus** [comment]
> decided setting the Conversions.State value to 14 (converted_unknown) 

Probably harmless. 

You will have noticed conversion state validation in the code when the program starts. As long as that validation does not conclude some nonsense you should be OK.
